services/frontend-service/src/assets/_variables.scss

```diff
@@ -5,6 +5,18 @@ $nav-list-item-padding: 3em;
 $nav-indicator-width: 8px;
 $nav-indicator-border-radius: 0px 5px 5px 0px;
 
+//Sidebar variables
+$sidebar-width: 30em;
+$sidebar-height: 95vh;
+$sidebar-header-button-margin: 5em;
+$sidebar-header-title-rpadding: 9em;
```

Comment:
what is `rpadding`?

Reply:
rapdding is a variable to alignt the header title 'Planned Actions' with the content of the side bar. Because the ehader title is composed of two divs positioned using flex. Using the text-align on the header text would not be enough to achieve the center alignment with the whole side bar.

Reply:
@tameremad I changed the variable name according to the variable naming, to be more descriptive of how it is supposed to be used